UnityProject/Assets/Scripts/Player/PlayerScript.cs

```diff
@@ -50,7 +50,7 @@ public class PlayerScript : NetworkBehaviour, IAdminInfo, IPlayerPossessable, IH
 	/// <summary>
 	/// Current character settings for this player.
 	/// </summary>
-	[SyncVar] public CharacterSheet characterSettings = new CharacterSheet();
+	[SyncVar, NonSerialized] public CharacterSheet characterSettings = new CharacterSheet();
```

Comment:
doesn't this prevent mobs from having their character sheets saved by the map saver?

Reply:
for mid round this is a problem but you're supposed to be using the blueprint if you want to spawn in a mob

Reply:
> for mid round this is a problem but you're supposed to be using the blueprint if you want to spawn in a mob

does the map loader automatically generate those blueprints for MobV2s? Because otherwise, this means that anyone who maps in newer mobs with specific info (like how they look or what they're called) will have missing data.

Reply:
for mid round saving  it doesn't replace them but that's not really supported too much,
if you're mapping you're supposed to use the blueprints for specifying what spawns